tests/km_core_tests.bats

```diff
@@ -1101,7 +1101,8 @@ fi
    KM_MGTPIPE=$MGMTPIPE km_with_timeout hello_html_test$ext $snapshot_test_port &
    # Wait for the process to exist
```

Comment:
Might want to change the comment to say process and mgmt pipe to exist.

Reply:
good point. dead code needed to be removed too.